src/Template/Element/sentences/transcription_button.ctp

```diff
@@ -0,0 +1,13 @@
+<div class="menu-wrapper" ng-if="vm.hasHiddenTranscriptions">
+    <div class="menu" layout="row" layout-align="space-between center">
+        <md-button ng-if="!vm.isMenuExpanded" class="md-icon-button" ng-click="vm.toggleMenu()">
+            <md-icon>unfold_more</md-icon>
+            <md-tooltip><?= __('Show all transcriptions') ?></md-tooltip>
```

Comment:
This message is used both in the context of multiple transcriptions (like Chinese) as well as single transcription (like Japanese). It would be best to have a pluralized message, but in the mean time, it may be better not to put so much emphasis on the plural by just saying "Show transcriptions" instead.

Reply:
Or maybe "Show unreviewed transcriptions", to match the opposite message for unfolding.